docs/proposals/20220927-label-sync-between-machine-and-nodes.md

```diff
@@ -21,28 +25,28 @@ Refer to the [Cluster API Book Glossary](https://cluster-api.sigs.k8s.io/referen
 
 ## Summary
 
-This document discusses a means by which labels placed on Machine can be kept in sync with corresponding labels on the underlying Kubernetes Nodes.
+This document discusses how labels placed on a Machine can kept in sync with the corresponding Kubernetes Node.
 
 ## Motivation
 
-Managing labels on Kubernetes nodes is a long standing [issue](https://github.com/kubernetes-sigs/cluster-api/issues/493) in Cluster API.
+Managing labels on Kubernetes nodes has been a long standing [issue](https://github.com/kubernetes-sigs/cluster-api/issues/493) in Cluster API.
 
-During [previous attempt](https://github.com/kubernetes-sigs/cluster-api/pull/6255) to address this issue (thanks @Arvinderpal for all the work) we discovered that this topic has several implications:
+The following challenges have been identified through various iterations:
 
-- Define how labels propagate from Machine to node
-- Define how labels propagate from ClusterClass to KCP/MachineDeployments/Machine Pools and down to Machines
-- Define how to prevent that label propagation triggers un-necessary rollouts
+- Define how labels propagate from Machine to node.
+- Define how labels propagate from ClusterClass to KubeadmControlPlane/MachineDeployments/Machine Pools, and ultimately to Machines.
+- Define how to prevent that label propagation triggers unnecessary rollouts.
 
 With the "divide and conquer" principle in mind this proposal aims to address the first point only, while the remaining points are going to be addressed in separated, complementary efforts.
 
 ### Goals
 
-- Support label synchronizing from Machine to the corresponding Kunernetes nodes for the labels that fall under the Kubernetes `node-role.kubernetes.io/` and `node-restriction.kubernetes.io/` prefix
-- Support label synchronizing from Machine to the corresponding Kunernetes nodes for the labels that fall under the Cluster API `node.cluster.x-k8s.io/` prefix
+- Support label sync from Machine to the linked Kubernetes node, limited to `node-role.kubernetes.io/` prefix and the `node-restriction.kubernetes.io` domain.
```

Comment:
For these ones, support additive on creation only sync?

Reply:
If we get SSA to work we can sync them continuously (which I would prefer as it's simpler and more intuitive to understand)